base/util.jl

```diff
@@ -326,17 +326,18 @@ function with_output_color(f::Function, color::Union{Int, Symbol}, io::IO, args.
         if !iscolor
             print(io, str)
         else
+            bold && color == :bold && (color = :nothing)
+            enable_ansi  = get(text_colors, color, text_colors[:default]) *
+                               (bold ? text_colors[:bold] : "")
+            disable_ansi = get(disable_text_style, color, text_colors[:default]) *
+                               (bold ? disable_text_style[:bold] : "")
```

Comment:
Does it matter if you don't end in the opposite order?  i.e. if you do `red*bold` does it need to end with `disablebold*defaultcolor`?

Reply:
It doesn't matter but changed it anyway. It seems nicer.